SportsResults.frockett/SportsResults.frockett/SportsResultsService.cs

```diff
@@ -0,0 +1,48 @@
+﻿using Microsoft.Extensions.Hosting;
+
+namespace SportsResults.frockett;
+
+internal class SportsResultsService : BackgroundService
+{
+    private readonly EmailService emailService;
+    private readonly ScraperService scraperService;
+
+    public SportsResultsService(EmailService emailService, ScraperService scraperService)
+    {
+        this.emailService = emailService;
+        this.scraperService = scraperService;
+    }
+
+    protected override async Task ExecuteAsync(CancellationToken stoppingToken)
+    {
+        DateTime startTime = DateTime.Now;
+        DateTime sendTime = new DateTime(startTime.Year, startTime.Month, startTime.Day, 16, 15, 0);
+        if (startTime > sendTime)
+        {
+            sendTime = sendTime.AddDays(1);
+        }
+
+        TimeSpan startupDelay = sendTime - startTime;
+        await Task.Delay(startupDelay, stoppingToken);
+
+        try
+        {
+            while (!stoppingToken.IsCancellationRequested)
+            {
+                var results = await scraperService.GetScoresAsync();
+                var table = EmailComposer.BuildHtmlTable(results);
+                EmailService.SendEmail(table);
```

Comment:
Email services injected but SendMail is a static method so there's no need to inject it :sweat_smile:  Unless you want to use the email service instance then make SendMail non static

![Screenshot from 2024-03-27 12-43-15](https://github.com/TheCSharpAcademy/CodeReviews.Console.SportsResults/assets/57032138/0420950c-f8a4-4012-a067-ee319b4a408f)

Reply:
Removed it from the fields and constructor